src/main/kotlin/uk/gov/justice/digital/hmpps/courtregister/CourtRegisterApplication.kt

```diff
@@ -2,8 +2,9 @@ package uk.gov.justice.digital.hmpps.courtregister
 
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.runApplication
+import org.springframework.cloud.aws.autoconfigure.messaging.MessagingAutoConfiguration
 
-@SpringBootApplication
+@SpringBootApplication(exclude = [MessagingAutoConfiguration::class])
```

Comment:
exclude the cloud aws auto configuration as we use HMPPS SQS stuff to create the aws clients